include/baresip.h

```diff
@@ -255,17 +255,18 @@ struct call  *call_find_linenum(const struct list *calls, uint32_t linenum);
 struct call  *call_find_id(const struct list *calls, const char *id);
 void call_set_current(struct list *calls, struct call *call);
 const struct list *call_get_custom_hdrs(const struct call *call);
-int call_set_media_direction(struct call *call, enum sdp_dir a,
+void call_set_media_direction(struct call *call, enum sdp_dir a,
```

Comment:
is it possible to split up the patch into logical blocks ?

Reply:
I tried to split it up a bit and got rid of one of the commits.